.github/workflows/antsibull-docs.yml

```diff
@@ -59,28 +59,25 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install poetry
-          sed -i -e 's/^python = .*/python = "^${{ matrix.python }}"/' pyproject.toml
-          poetry install
-          poetry update
+          pip install -e .[coverage] ../antsibull-core
```

Comment:
```suggestion
          pip install .[coverage] ../antsibull-core
```

Reply:
I prefer editable installs when doing code coverage. Otherwise coverage reports could be a mess since random paths are in there.